lib/project_types/script/messages/messages.rb

```diff
@@ -47,41 +47,38 @@ module Messages
           invalid_language_cause: "Invalid language %s.",
           invalid_language_help: "Allowed values: %s.",
 
-          missing_script_config_yml_field_cause: "The script.config.yml file is missing the required %s field.",
```

Comment:
@MeredithCastile there have been some content changes here. It'd be great to get your thoughts on those.

Reply:
With the exception of the added messages (`configuration_error_cause` and `configuration_error_help`), all of the messages should be the same, just with placeholders for the file name during this grace period when we support both `script.config.yml` and `script.json`, and to make it easier to change the filename in the future (not hard-coded in a bunch of strings).